pkg/model/iam/iam_builder.go

```diff
@@ -219,10 +219,6 @@ func (b *PolicyBuilder) BuildAWSPolicyNode() (*Policy, error) {
 		addECRPermissions(p)
 	}
 
-	if b.Cluster.Spec.Networking != nil && b.Cluster.Spec.Networking.AmazonVPC != nil {
-		addAmazonVPCCNIPermissions(p, resource, b.Cluster.Spec.IAM.Legacy, b.Cluster.GetName())
```

Comment:
Did you mean to delete this? It should probably be included in the Networking Spec checks lower.